uefi-raw/src/table/runtime.rs

```diff
@@ -98,6 +98,17 @@ newtype_enum! {
 
         /// A platform-specific reset type.
         PLATFORM_SPECIFIC = 3,
+
+        /// The reset type is undefined, signifying
+        /// an error has occurred when attempting to
+        /// query a reset type.
+        UNDEFINED = 4,
```

Comment:
Is this in the spec? I don't see it at https://uefi.org/specs/UEFI/2.10/08_Services_Runtime_Services.html?highlight=reset_type#reset-system